kubeadm/scripts/Install-Containerd.ps1

```diff
@@ -11,20 +11,38 @@ This script
 
 .PARAMETER ContainerDVersion
 ContainerD version to download and use.
+For HostProcess containers minimal version is 1.6.0.
 
 .PARAMETER netAdapterName
 Name of network adapter to use when configuring basic nat network.
 
+.PARAMETER UseHostProcess
+Declares that HostProcess model is in use. Some code should be omitted.
+
 .EXAMPLE
 PS> .\Install-Conatinerd.ps1
 
+.EXAMPLE
+PS> .\Install-Containerd.ps1 -UseHostProcess -ContainerDVersion # 1.6.0+
+
 #>
 
+[CmdletBinding(
+    DefaultParameterSetName="kubeadm"
+)]
 Param(
     [parameter(HelpMessage = "ContainerD version to use")]
     [string] $ContainerDVersion = "1.6.8",
-    [parameter(HelpMessage = "Name of network adapter to use when configuring basic nat network")]
-    [string] $netAdapterName = "Ethernet"
+    [parameter(
+        HelpMessage = "Name of network adapter to use when configuring basic nat network",
+        ParameterSetName = "kubeadm"
+    )]
+    [string] $netAdapterName = "Ethernet",
```

Comment:
I think we can drop the `$netAdapterName` param since this script no longer creates a nat network (removed with https://github.com/kubernetes-sigs/sig-windows-tools/pull/254 b/c it was causing lots of issues)

Reply:
Yes, now it is unused parameter. Should we take care of the backward compatibility? If not, then we can do some more cleanups in both scripts. If so, these scripts will not fit kubeadm folder no longer. Maybe better/cleaner version of the scripts should be made in host-process folder. But there is a couple of PR's which makes changes to them right now. It is not a good time, I think.

I am not convinced to update referenced containerd version in this PR.